Makefile

```diff
@@ -108,7 +108,7 @@ applyTemplate:
 	oc process -f $(WHAT) | oc apply -f -
 .PHONY: applyTemplate
 
-postsubmit-update: origin-release origin-stable
+postsubmit-update: origin-release
```

Comment:
They are already on app.ci (not sure which postubmit does that, but there is one). They don't actually build there beause of the missing push secret, so that needs fixing first

Reply:
The postsubmit `branch-ci-openshift-release-master-config-updates` applies it to `app.ci`.
It will be removed after the card.

I will fix it on build01.
https://issues.redhat.com/browse/DPTP-1835?focusedCommentId=15638572&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-15638572